contrib/observability_ready_util/src/main/java/io/opencensus/contrib/observability/ready/util/BasicSetup.java

```diff
@@ -0,0 +1,113 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.observability.ready.util;
+
+import io.opencensus.common.ExperimentalApi;
+import io.opencensus.contrib.grpc.metrics.RpcViews;
+import io.opencensus.exporter.metrics.ocagent.OcAgentMetricsExporter;
+import io.opencensus.exporter.metrics.ocagent.OcAgentMetricsExporterConfiguration;
+import io.opencensus.exporter.trace.ocagent.OcAgentTraceExporter;
+import io.opencensus.exporter.trace.ocagent.OcAgentTraceExporterConfiguration;
+import io.opencensus.trace.Tracing;
+import io.opencensus.trace.config.TraceConfig;
+import io.opencensus.trace.config.TraceParams;
+import io.opencensus.trace.samplers.Samplers;
+
+/**
+ * Setup class to enable OpenCensus stats and metrics collections easily.
+ *
+ * @since 0.25
+ */
+@ExperimentalApi
+public final class BasicSetup {
+  private static final double DEAFULT_SAMPLING_RATE = 0.0001;
+  private static final String DEAFULT_ENDPOINT = "localhost:55678";
+
+  private BasicSetup() {}
+
+  /**
+   * Enables OpenCensus metric and traces.
+   *
+   * <p>This will register all basic {@link io.opencensus.stats.View}s. When coupled with an agent,
+   * it allows users to monitor application behavior.
+   *
+   * <p>Please note that in addition to calling this method, the application must:
+   *
+   * <ul>
+   *   <li>Include opencensus-contrib-observability-ready-util dependency on the classpath
+   *   <li>Deploy the OpenCensus agent
+   * </ul>
+   *
+   * <p>Example usage for maven:
+   *
+   * <pre>{@code
+   * <dependency>
+   *   <groupId>io.opencensus</groupId>
+   *   <artifactId>opencensus-contrib-observability-ready-util</artifactId>
+   *   <version>${opencensus.version}</version>
+   * </dependency>
+   * }</pre>
+   *
+   * <p>It is recommended to call this method before doing any RPC call to avoid missing stats.
+   *
+   * <pre>{@code
+   * BasicSetup.enableOpenCensus("with-service-name");
+   * }</pre>
+   *
+   * @param serviceName the service name.
+   * @param endPoint the end point of OC-Agent.
+   * @param probability the desired probability of sampling. Must be within [0.0, 1.0].
+   * @since 0.25
+   */
+  public static void enableOpenCensus(String serviceName, String endPoint, double probability) {
+    // register basic rpc views
+    RpcViews.registerAllGrpcBasicViews();
+
+    // set sampling rate
+    TraceConfig traceConfig = Tracing.getTraceConfig();
+    TraceParams activeTraceParams = traceConfig.getActiveTraceParams();
+    traceConfig.updateActiveTraceParams(
+        activeTraceParams.toBuilder().setSampler(Samplers.probabilitySampler(probability)).build());
+
+    // create and register Trace Agent Exporter
+    OcAgentTraceExporter.createAndRegister(
+        OcAgentTraceExporterConfiguration.builder()
```

Comment:
What about we add this in the ocagent artifact?

Reply:
As discussed, will open separate PR to show how this basic setup looks like in the exporter. 
Btw, ocagent has multiple artifacts, [contrib-agent](https://github.com/census-instrumentation/opencensus-java/tree/master/contrib/agent) artifact and ocagent [exporter](https://github.com/census-instrumentation/opencensus-java/tree/master/exporters/trace/ocagent). I think you are referring to exporter one, can you pls confirm.

Reply:
Alternative approach: https://github.com/census-instrumentation/opencensus-java/pull/2003

Reply:
I had a discussion with @rghetia and @dinooliva, we think this `opencensus-contrib-observability-ready-util` artifact is much more meaningful and useful than the exporter wise basic setup function.

1. Exporter level function would create confusion the users.
2. Exporter level function is just reducing 1-2 line of code i.e. enabling the views or sampling rate.
3. It looks weird to enable views using exporter level setup function.
4. We can start with this util and add same thing for SD or other exporters in the future, based on the adoption and usage pattern.
5. Mostly this `opencensus-contrib-observability-ready-util` will be used by Single click deployment, so makes sense to have agent exporter by default.

@rghetia and @dinooliva please add more, if anything is incorrect or missing.

@bogdandrutu WDYT?